.gitignore

```diff
@@ -5,3 +5,6 @@ node_modules
 npm-debug.log
 # WebStorm IDE files
 .idea/*
+*.iml
+
+*swp
```

Comment:
ok, just I'm not a vim user (still no idea how to exit it :) ), so I was wondering what it's about